rest_api/sawtooth_rest_api/rest_api.py

```diff
@@ -177,11 +195,26 @@ def main():
                   " host:port".format(rest_api_config.bind[0]))
             sys.exit(1)
 
+        reporter = None
+        if rest_api_config.opentsdb_url:
+            url = urlparse(rest_api_config.opentsdb_url)
+            proto, db_server, db_port, = url.scheme, url.hostname, url.port
+
+            registry = MetricsRegistry()
+            reporter = InfluxReporter(
+                registry=registry,
+                reporting_interval=1,
```

Comment:
This reporting interval is probably a bit aggressive. We're doing every 10 seconds on the telegraf config.